module/exporter-shopware-6/src/Infrastructure/Connector/Shopware6Connector.php

```diff
@@ -90,10 +95,16 @@ private function request(Shopware6Channel $channel, ActionInterface $action)
      */
     private function requestToken(Shopware6Channel $channel): void
     {
-        $post = new PostAccessToken($channel);
-        $data = $this->request($channel, $post);
-        $this->token = $data['access_token'];
-        $this->expiresAt = $this->calculateExpiryTime((int) $data['expires_in']);
+        try {
+            $post = new PostAccessToken($channel);
+            $data = $this->request($channel, $post);
+            $this->token = $data['access_token'];
+            $this->expiresAt = $this->calculateExpiryTime((int) $data['expires_in']);
+        } catch (ClientException $exception) {
+            if ($exception->getCode() === 401) {
```

Comment:
Shouldn't we somehow catch other exception as well, if happen? Otherwise, it'd be unable to authorize further on. wdyt?